login-service/src/strategies/StrategyUsingPassport.ts

```diff
@@ -71,7 +71,7 @@ export abstract class StrategyUsingPassport extends Strategy {
                 passportStrategy,
                 {
                     session: false,
-                    state: jwtService.sign(authStateData),
+                    state: jwtService.sign(authStateData), // TODO: check if an expiration and/or an additional random value are needed
```

Comment:
I assume the TODO comment is on intention. Perhaps also create an issue for it so we do not miss the TODO?

Reply:
done